dips/dip-1.mdx

```diff
@@ -1,6 +1,6 @@
 ---
 dip: 1
-title: Off-chain API
+title: Off-Chain API
```

Comment:
the title here is a little funny, is it Off-Chain Protocol and Travel Rule Exchange, not sure where the "API" is except maybe TR... I digress